pullrequest.js

```diff
@@ -55,6 +62,7 @@ function getId(path) {
 }
 
 function toggleDiff(id, duration, display) {
+    var prId = getCurrentPrId();
```

Comment:
In my version of this fix, I set some global (to this script) variables in the same loop as we inject the HTML. This way we can avoid doing the call to jQuery every time we toggle a diff, since the prId will only change as often as we need to inject HTML.